content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,670 @@
+    - configmaps
+  verbs: ['get']
+- apiGroups:
+    - discovery.k8s.io
+  resources:
+    - endpointslices
+  verbs: ['get', 'list', 'watch']
+- apiGroups:
+    - networking.k8s.io
+  resources:
+    - ingresses
+  verbs: ['get', 'list', 'watch']
+- nonResourceURLs: ['/metrics']
+  verbs: ['get']
+```
+
+If you enable the `prometheusCR` (set
+`spec.targetAllocator.prometheusCR.enabled` to `true`) in the
+OpenTelemetryCollector CR, you will also need to define the following roles.
+These give the Target Allocator access to the PodMonitor and ServiceMonitor CRs.
+It also gives namespace access to the PodMonitor and ServiceMonitor.
+
+```yaml
+- apiGroups:
+    - monitoring.coreos.com
+  resources:
+    - servicemonitors
+    - podmonitors
+  verbs:
+    - '*'
+- apiGroups: ['']
+  resources:
+    - namespaces
+  verbs: ['get', 'list', 'watch']
+```
+
+## Additional OTel components for Kubernetes
+
+This section covers additional OTel Collector components you can use to capture
+Kubernetes metrics.
+
+Receiving data:
+
+- [Kubernetes Cluster Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver):
+  collects cluster-level metrics and entity events from the
+  [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
+- [Kubernetes Objects Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver):
+  collects (pull/watch) objects from the Kubernetes API server
+- [Kubelet Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver):
+  pulls node, pod, container, and volume metrics from the API server on a
+  [Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)
+  and sends it down the metric pipeline for further processing
+- [Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver):
+  scrapes system metrics from hosts that make up the cluster
+
+Processing data:
+
+- [Kubernetes Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor):
+  adds Kubernetes context, thereby
+- enabling you to correlate application telemetry with your Kubernetes telemetry
+- – considered one of the most important components for monitoring Kubernetes
+- with OpenTelemetry
+
+You can also use the Kubernetes attributes processor to set custom resource
+attributes for traces, metrics, and logs using the Kubernetes labels and
+annotations you’ve added to your pods and namespaces.
+
+There are a few more Collector components you can implement to monitor
+Kubernetes, including Kubernetes-specific ones as well as general-use
+processors, such as the batch, memory limiter, and resource processors. You can
+read more about them [here](docs/kubernetes/collector/components/).
```

Comment:
The link path is missing a leading `/` (it is invalid otherwise). Let's rework this to avoid the "here" link text:

```suggestion
processors, such as the batch, memory limiter, and resource processors. To learn
more, see [Important Components for
Kubernetes](/docs/kubernetes/collector/components/).
```